build.xml

```diff
@@ -1111,6 +1107,34 @@ cd ..
     <copy todir="${lib.native}">
       <fileset dir="${build}" includes="${native.jar}"/>
     </copy>
+    <condition property="is_darwin">
+      <matches string="${os.prefix}" pattern="^darwin-"/>
+    </condition>
+    <mkdir if:set="is_darwin" dir="${build}/native-darwin-universal/intel"/>
+    <mkdir if:set="is_darwin" dir="${build}/native-darwin-universal/aarch64"/>
+    <copy if:set="is_darwin" todir="${build}/native-darwin-universal/intel">
+      <zipfileset src="${lib.native}/darwin-intel.jar" includes="*jnidispatch*"/>
+    </copy>
+    <copy if:set="is_darwin" todir="${build}/native-darwin-universal/aarch64">
+      <zipfileset src="${lib.native}/darwin-aarch64.jar" includes="*jnidispatch*"/>
+    </copy>
+    <exec if:set="is_darwin" executable="lipo" dir="${build}/native-darwin-universal" failonerror="true">
+      <arg value="-create"/>
+      <arg value="-output"/>
+      <arg value="libjnidispatch.jnilib"/>
+      <arg value="intel/libjnidispatch.jnilib"/>
+      <arg value="aarch64/libjnidispatch.jnilib"/>
+    </exec>
+    <jar if:set="is_darwin" jarfile="${build}/darwin.jar" createUnicodeExtraFields="never" encoding="UTF-8">
+      <fileset dir="${build}/native-darwin-universal" includes="libjnidispatch.jnilib"/>
+      <manifest>
+        <attribute name="Implementation-Version" value="${jni.version} b${jni.build}"/>
+        <attribute name="Specification-Version" value="${jni.version}"/>
+      </manifest>
+    </jar>
+    <copy if:set="is_darwin" todir="${lib.native}">
+      <fileset dir="${build}" includes="darwin.jar"/>
+    </copy>
```

Comment:
@matthiasblaesing This might be a bit crude. Please advise, if you'd prefer this e.g. as a `Makefile`, or if you have suggestions how to do this nicer in ant.